astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -30,6 +28,97 @@
 # prevent having to create new Angle objects every time `get_roll0` is called.
 _ROLL0 = Angle(58.5986320306*u.degree)
 
+
+class default_sun_galactocentric(ScienceState):
+    """The ScienceState instance that controls global setting of Galactocentric
+    default solar motion parameters
+    """
```

Comment:
Will want to elaborate a bit on this...  In particular, on how to initialize it.